internal/cli/universal_login_customize.go

```diff
@@ -326,6 +332,41 @@ func (h *webSocketHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.shutdown()
 		return
 	}
+
+	for {
+		var message webSocketMessage
+		if err := connection.ReadJSON(&message); err != nil {
+			h.display.Errorf("failed to read WebSocket message: %v", err)
+			continue
+		}
+
+		switch message.Type {
+		case "fetch_prompt":
+			var promptToFetch promptData
+			if err := json.Unmarshal(message.Payload, &promptToFetch); err != nil {
+				h.display.Errorf("failed to unmarshal %q payload: %v", message.Type, err)
+				continue
```

Comment:
For now opting not to close the connection, but this will most likely be subject to change when we start integrating with the web app in a closer manner.